src/gasman.c

```diff
@@ -1883,88 +1875,6 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * sweep phase * * * * * * * * * * * * * * */
 
-#if 0
-    /* call freeing function for all dead bags                             */
-    if ( NrTabFreeFuncBags ) {
```

Comment:
This code was addd in the initial import of the CVS repository (!) It seems to be a "finalizer" API. As such, we may want to keep it for PR #57 -- that one looks to me as if @rbehrends actually reused the `FreeFuncBag` staff...

But *if* we remove this then, then we should also remove all the related obsolete things, i.e., anything which contains `FreeFuncBag`, such as:
* `NrTabFreeFuncBags`
* `InitFreeFuncBag`
* `TNumFreeFuncBags`